zookeeper-server/src/main/java/org/apache/zookeeper/ClientCnxn.java

```diff
@@ -616,6 +616,14 @@ private void processEvent(Object event) {
                       } else {
                           cb.processResult(rc, clientPath, p.ctx, null);
                       }
+                  } else if (p.response instanceof GetAllChildrenNumberResponse) {
```

Comment:
Also need to import GetAllChildrenNumberResponse

Reply:
> Also need to import GetAllChildrenNumberResponse

OK, thanks for your remind.